2_gitlab/cloudbuild-destroy.yaml

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s # 1-hr
+tags:
+  - modern-cicd-anthos
+  - gitlab
+substitutions:
+  _DOMAIN: ""
+  _BUILD_IMAGES: "true"
+  _CREATE_GITLAB: "true"
+  _DESTROY_CLUSTER: "true"
+  _DESTROY_REPOS: "true"
+options:
+  substitution_option: 'ALLOW_LOOSE'
+steps:
+# Uninstall gitlab additions
+- name: 'gcr.io/${PROJECT_ID}/anthos-platform-installer'
+  id: 'destroy-gitlab-install'
```

Comment:
IMO the config of `repos` as a separate subdirectory under `2_gitlab` has always been confusing. I think we should either fold it directly into the main `2_gitlab` config or move them into a new top-level component.

@viglesiasce Thoughts?

Reply:
@morgante and @viglesiasce can we address this proposed change in a future CL/PR so we keep the scope just to restructuring?

For this discussion, what if we say "instead of "2_gitlab", we go to "2_cicd", which has multiple folders (or call them modules) that would be "gitlab", "github", "jenkins"...then the folder for repos could either be under "gitlab" or in the same folder (more brainstorming for extended module breakup than this one commnet...want to explore other packaging options)

Reply:
Agreed, we can do this in a separate issue. https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/issues/51 to track

Reply:
Yeah for sure. I dont recall why I did that way.